playtime/init.lua

```diff
@@ -53,15 +53,24 @@ local function format_duration(seconds)
 end
 
 minetest.register_chatcommand("playtime", {
-	params = "",
+	params = "<player_name>",
```

Comment:
This should be `[playername]` because it's optional, and people tend to say `playername` instead of `player_name`